app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -0,0 +1,61 @@
+require 'set'
+
+module Katello
+  module Pulp
+    module PulpContentUnit
```

Comment:
Honestly i think it was just done that way as that is how pulp_database_unit was done.  I can change it if preferred.  It probably does make more sense